package.json

```diff
@@ -40,7 +40,7 @@
     "lint:format": "prettier --check . --ignore-path .gitignore",
     "semantic-release": "semantic-release",
     "messages:extract": "lingui extract",
-    "messages:compile": "lingui compile --strict"
+    "messages:compile": "lingui compile"
```

Comment:
Should we remove the `--strict` option forever, or should it be restored when all translations are in place?

Reply:
@imprashast If I understood correctly from our chat yesterday, we do not need --strict option anymore? :)